### PR TITLE
Diversity tickets

### PR DESCRIPTION
**Pull request by nikgraf**

### CODE REVIEW

File: src/pages/coc.re [hunk 1]
```diff
@@ -16,14 +17,43 @@ let make = (_children) => {
...
+        </div>
+      </p>
       <h2> (ste("The Quick Version")) </h2>
       <p>
         (
           ate([|
-            (ste("Our conference is dedicated to providing a harassment-free conference experience for everyone, regardless of gender, gender identity and expression, age, sexual orientation, disability, physical appearance, body size, race, ethnicity, religion (or lack thereof), or technology choices. We do not tolerate harassment of conference participants in any form. Sexual language and imagery is not appropriate for any conference venue, including talks, workshops, parties, Twitter and other online media. Conference participants violating these rules may be sanctioned or expelled from the conference ")),
-            <em>(ste("without a refund"))</em>,
-            (ste(" at the discretion of the conference organisers.")),
+            ste(
```
Comment (okonet) [inline src/pages/coc.re]: Consider using Markdown for this instead. See https://github.com/ReasonVienna/reason-conf/blob/master/src/pages/sponsors.re#L40

Comment (nikgraf) [reply src/pages/coc.re]: Can we do this in another PR? This was just reformatting from rfmt

Comment (ryyppy) [reply src/pages/coc.re]: I refactored it

**Comment by nikgraf** [comment]
@ryyppy ohhh, now I see what @okonet meant. Thanks for fixing my stuff

**Pull request merged**